Plugins/Public/base_plugin/Main.cpp

```diff
@@ -2775,6 +2823,46 @@ void Plugin_Communication_CallBack(PLUGIN_MESSAGE msg, void* data)
 			base->Save();
 		}
 	}
+	else if (msg == CUSTOM_BASE_LAST_DOCKED)
+	{
+		LAST_PLAYER_BASE_NAME_STRUCT* info = reinterpret_cast<LAST_PLAYER_BASE_NAME_STRUCT*>(data);
+		if (clients.count(info->clientID))
```

Comment:
I am not familiar with .count method. How is this different from .find? Surely find is more efficient as it stops after first match.

Reply:
It's the c++14's equivalent of .contains(). For maps it only ever returns 0 or 1 as per [documentation](https://cplusplus.com/reference/map/map/count/). I would _assume_ it's somewhat optimized.